src/central_freelist.cc

```diff
@@ -143,8 +143,7 @@ void CentralFreeList::ReleaseToSpans(void* object) {
     }
     lock_.Lock();
   } else {
-    *(reinterpret_cast<void**>(object)) = span->objects;
-    span->objects = object;
+    FL_Push(&(span->objects), object);
```

Comment:
We need to keep "abstraction" for set of elements in central free distinct from thread caches. I actually plan to merge google-internal changes somewhat soon that change format for central freelist.

Reply:
So, no doubly linked list for the central cache?
Should I just revert all changes to this file?

Reply:
The doubly linked list is used for the objects inside a span. The list of spans is its own abstraction.
Do you want me to revert the use of doubly-linked list for span objects?